tini/src/seccomp_fd_notify.c

```diff
@@ -248,8 +274,16 @@ static int install_notify_filter(void) {
 	/* Only one listening file descriptor can be established. An attempt to
 	   establish a second listener yields an EBUSY error. */
 
+	if (get_kernel_version(&system_kernel_version) == -1) {
+		return -1;
+	}
+
+	if (system_kernel_version >= WAIT_KILLABLE_SECCOMP_KERNEL_VERSION) {
```

Comment:
Can you just >= on a tuple of integers like that? How does C do a comparison with `(5, 15, 35)` ?

Reply:
> Can you just >= on a tuple of integers like that? How does C do a comparison with `(5, 15, 35)` ?

So for this macro `#define WAIT_KILLABLE_SECCOMP_KERNEL_VERSION KERNEL_VERSION(5, 15, 35)` `KERNEL_VERSION` expands to `(((a) << 16) + ((b) << 8) + (c))`